packages/components/src/components/Text/Text.tsx

```diff
@@ -0,0 +1,130 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { type ReactNode, forwardRef, type ForwardedRef, type HTMLAttributeAnchorTarget } from 'react';
+
+import { type CommonAriaProps } from '#/helpers/aria';
+import { cn } from '#/utilities/styleUtilities';
+
+import styles from './styles/text.module.scss';
+
+type TextWeight = 'default' | 'strong' | 'x-strong';
+type TextSize = 'x-small' | 'small' | 'medium' | 'large';
+type TextColor = 'default' | 'weak' | 'x-weak' | 'disabled' | 'negative' | 'positive' | 'warning' | 'interactive';
+type BoxColor = 'neutral' | 'selected' | 'disabled' | 'positive' | 'negative' | 'warning';
+
+type TagType = 'a' | 'abbr' | 'address' | 'em' | 'label' | 'li' | 'span' | 'strong' | 'time' | 'p';
+
+export type TextProps<TTag extends TagType = 'span'> = CommonAriaProps &
+    TagProps<TTag> & {
+        /**
+         * `id` represents the id of the element
```

Comment:
Keep the same structure to the docs 😉 
We know it is the `id`, we are hovering it.

```suggestion
         * The id of the element
```

Reply:
Well, it's called id... duh.😂